SafeguardSessions/CHANGELOG.md

```diff
@@ -0,0 +1,31 @@
+# Changelog
+
+## [Unreleased]

```

Comment:
#Optional: I feel adding this subchapter should be part of final release.

Reply:
Basically the unreleased part will be the contents of the new release notes.

With unreleased, the changelog needs to be modified in every PR. Is it worth the extra overhead?

Reply:
I think it is easier to have an Unreleased section beforehand and fill up the subsections with modifications as we are merging them because you are more in "context" of the modification and you can phrase the corresponding release note summary easier and quicker. 

Looking back at the git history release time, filtering commits and writing the release note summaries takes more effort in my opinion.

This way we could shorten the time we spend on putting the release notes together.

But if all of you insist on it, I can remove the Unreleased section.

Reply:
It can stay. ;)